src/rpc/core_rpc_server.cpp

```diff
@@ -2136,9 +2136,9 @@ namespace cryptonote
   bool core_rpc_server::on_get_coinbase_tx_sum(const COMMAND_RPC_GET_COINBASE_TX_SUM::request& req, COMMAND_RPC_GET_COINBASE_TX_SUM::response& res, epee::json_rpc::error& error_resp, const connection_context *ctx)
   {
     PERF_TIMER(on_get_coinbase_tx_sum);
-    std::pair<uint64_t, uint64_t> amounts = m_core.get_coinbase_tx_sum(req.height, req.count);
-    res.emission_amount = amounts.first;
-    res.fee_amount = amounts.second;
+    std::tuple<uint64_t, uint64_t, uint64_t> amounts = m_core.get_coinbase_tx_sum(req.height, req.count);
+    res.emission_amount = std::get<0>(amounts);
+    res.fee_amount = std::get<1>(amounts);
```

Comment:
Should add the burn amount here as well.

Otherwise PR looks good to me.

Reply:
Also a bit of modern C++ coolness can let you write it as:
```C++
std::tie(res.emission_amount, res.fee_amount, res.burn_amount) =
    m_core.get_coinbase_tx_sum(req.height, req.count);
```